.golangci.yml

```diff
@@ -98,3 +98,12 @@ issues:
     - path: models/update.go
       linters:
         - unused
+    - path: modules/setting/attachment.go
+      linters:
+        - dupl
+    - path: modules/setting/lfs.go
+      linters:
+        - dupl
+    - path: cmd/dump.go
+      linters:
+        - dupl
```

Comment:
Do we really need this?

This is a bad sign, we should be removing lint exceptions not adding them.

Reply:
I removed
```
- path: modules/setting/attachment.go
      linters:
        - dupl
    - path: modules/setting/lfs.go
      linters:
        - dupl
```
but still keep `cmd/dump.go` because it reports the duplicated codes but I think  merging them as a function is not a good choice.